package.json

```diff
@@ -27,29 +27,74 @@
 	],
 	"scripts": {
 		"postinstall": "npm run bootstrap",
-		"prettier:check": "prettier --check \"packages/*/src/**/*.ts\"",
 		"prettier:write": "prettier --write \"packages/*/src/**/*.ts\"",
 		"postpublish": "lerna run install:safe --scope lit-plugin && echo \"🎉 Published successfully! You can now publish the vscode extension seperately by running 'cd packages/vscode-lit-plugin && npm run publish'\"",
 		"publish": "lerna run prepare && lerna publish --exact && npm run postpublish",
 		"publish:next": "lerna run prepare && echo \"When asked, please choose 'Custom Prerelease' and use default value\" && lerna publish --dist-tag next --preid next --exact && npm run postpublish",
-		"bootstrap": "lerna clean --yes && lerna bootstrap --hoist --ignore lit-plugin && lerna run install:safe --scope lit-plugin && npm run copylink",
-		"lint": "eslint .",
-		"build": "lerna run build --stream",
-		"watch": "lerna run watch --parallel --stream",
-		"test": "lerna run test --parallel --stream",
-		"test:all": "lerna run test:all --parallel --stream",
+		"bootstrap": "lerna clean --yes && lerna bootstrap && lerna run install:safe --scope lit-plugin",
 		"test:watch": "lerna run test:watch --parallel --stream",
 		"readme": "lerna run readme && readme generate -i readme.blueprint.md -c readme.config.json",
 		"dev": "cd dev && TSS_DEBUG=5999 code . --disable-extension runem.lit-plugin",
 		"dev:logs": "touch dev/lit-plugin.log && tail -f dev/lit-plugin.log",
-		"copylink": "node scripts.js copylink",
-		"copylink:watch": "nodemon --exec \"npm run copylink\""
+		"lint": "wireit",
+		"build": "wireit",
+		"test": "wireit",
+		"test:headless": "wireit",
+		"test:headful": "wireit",
+		"copylink": "wireit",
+		"eslint": "wireit",
+		"prettier:check": "wireit"
+	},
+	"wireit": {
+		"build": {
+			"dependencies": [
+				"./packages/lit-analyzer:build",
+				"./packages/ts-lit-plugin:build",
+				"./packages/vscode-lit-plugin:build"
+			]
+		},
+		"test": {
+			"dependencies": [
+				"test:headless",
+				"test:headful",
+				"lint"
+			]
+		},
+		"lint": {
+			"dependencies": [
+				"eslint",
+				"prettier:check"
+			]
+		},
+		"test:headless": {
+			"dependencies": [
+				"./packages/lit-analyzer:test"
+			]
+		},
+		"test:headful": {
+			"dependencies": [
+				"./packages/vscode-lit-plugin:test"
+			]
+		},
+		"copylink": {
+			"dependencies": [
+				"./packages/lit-analyzer:build",
+				"./packages/ts-lit-plugin:build"
+			],
+			"command": "node scripts.js copylink"
+		},
+		"eslint": {
```

Comment:
This is fine, but just FYI these could also be regular old npm scripts, since they dont have any dependencies, files, output. Wireit scripts can depend on non-Wireit scripts.